### ISSUE TITLE
How many epochs needed to get the PQ on paper?

### ISSUE THREAD

**Issue by hamin-song** [robot-learning-freiburg/EfficientLPS#5]
Thank you for providing awesome network!

I'm training your model in the same SW environment as you mentioned. (OS, CUDA, PyTorch...)
My GPU is NVIDIA GeForce RTX 2080 Ti * 2.
And imgs_per_gpu is 1.

To get the PQ on paper (semantic KITTI validation set: **59.2**),
**How many epochs do I have to train?**

Now I'm currently training for the 33rd epoch, and the best val PQ so far is **44.6** of the 25th epoch.
It also appears to have almost converged.
